code/modules/ruins/lavalandruin_code/seed_vault.dm

```diff
@@ -0,0 +1,33 @@
+/obj/effect/spawner/lootdrop/seed_vault
+	name = "seed vault seeds"
+	lootcount = 1
+
+	loot = list(/obj/item/seeds/gatfruit = 10,
+				/obj/item/seeds/cherry/bomb = 10,
+				/obj/item/seeds/berry/glow = 10,
+				/obj/item/seeds/sunflower/moonflower = 8
+				)
+
+/obj/effect/mob_spawn/human/seed_vault
+	name = "preserved terrarium"
+	desc = "An ancient machine that seems to be used for storing plant matter. The glass is obstructed by a mat of vines."
+	mob_name = "a lifebringer"
+	icon = 'icons/obj/lavaland/spawners.dmi'
+	icon_state = "terrarium"
+	density = TRUE
+	roundstart = FALSE
+	death = FALSE
+	mob_species = /datum/species/diona
```

Comment:
I decided to just use Diona here. That does mean the flavour_text below may need a bit of a rewrite.